### PR TITLE
Bridge dynamic logging

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Enhancement / new feature

### Description
Allow setting bridge logging configuration without RU (dynamically).

### Checklist
- [x] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [x] Update CHANGELOG.md

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java [hunk 1]
```diff
@@ -349,6 +352,111 @@ void testDynamicallySetEOloggingLevels() throws InterruptedException {
...
+
+        kubeClient().getClient().configMaps().inNamespace(NAMESPACE).createOrReplace(configMapBridge);
+
+        ExternalLogging bridgeXternalLogging = new ExternalLoggingBuilder()
+                .withName("external-configmap-bridge")
+                .build();
+
+        LOGGER.info("Setting log level of Bridge to OFF - records should not appear in the log");
+        // change to the external logging
+        KafkaBridgeResource.replaceBridgeResource(CLUSTER_NAME, bridz -> {
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:440]: bridz?

Comment (stanlyDoge) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:440]: Yeah, it is a prevention of using test-scope-var and lambda-scope-var with the same name. I can rename it if you wish so.

Comment (ppatierno) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:440]: is that a czech word? :-)

**Comment by stanlyDoge** [comment]
@strimzi-ci run tests false profile=regression testcase=io.strimzi.systemtest.log.LoggingChangeST#testDynamicallySetBridgeLoggingLevels

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java (hunk 1 above)
Comment (stanlyDoge) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:440]: Kinda :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,7 @@
   * enable/disable metrics in the KafkaBridge custom resource
   * new Grafana dashboard for the bridge metrics
 * Support dynamically changeable logging in the Entity Operator 
+* Support dynamically changeable logging in the Kafka Bridge
```
Comment (ppatierno) [inline CHANGELOG.md:25]: maybe both on one line?

**Pull request merged**